style50/__main__.py

```diff
@@ -54,7 +54,7 @@ def main():
                         help="specify output mode. Valid modes are character (default), split, unified, score, and json")
     parser.add_argument("-v", "--verbose", action="store_true",
                         help="print full tracebacks of errors")
-    parser.add_argument("--version", action="version", version="%(prog)s {}".format(__version__))
+    parser.add_argument("-V", "--version", action="version", version="%(prog)s {}".format(__version__))
```

Comment:
`astyle -V` outputs `Artistic Style Version 3.0.1`, which seems unnecessarily verbose. And we're currently outputting `style50 x.y.z`, but just `x.y.z` seems sufficient?